index.bs

```diff
@@ -1783,7 +1783,7 @@ a numbered step. If outdented, it (today) is rendered either as a bullet in the
                         ::  |attestationObject|
 
                         :   {{AuthenticatorAttestationResponse/[[transports]]}}
-                        ::  A sequence of zero or more unique {{DOMString}}s, in lexicographical order, that the |authenticator| is believed to support. The values SHOULD be members of {{AuthenticatorTransport}}.
+                        ::  A sequence of zero or more unique {{DOMString}}s, in lexicographical order, that the |authenticator| is believed to support. The values SHOULD be members of {{AuthenticatorTransport}}, but [=client platforms=] MUST ignore unknown values.
```

Comment:
down below where `AuthenticatorAttestationResponse` interface is devined, the language for `[[transports]]` says: 
> The values SHOULD be members of AuthenticatorTransport but Relying Parties MUST **accept** unknown values.

Should we `s/ accept / ignore /` in the latter (it is presently untouched by this PR) or instead alter the above change to be:
> , but [=client platforms=] MUST **accept** unknown values.

..?

I.e., ought the language be the same in both places?

Reply:
It should indeed be the same in both, and I don't find myself having a definite preference, though I think `ignore unknown values` might be clearer. @agl?

Reply:
thx, `ignore unknown values` is fine by me, and we should make the language the same in both places.

Reply:
Fixed in https://github.com/w3c/webauthn/pull/1392/commits/8b0dbbc2cf7a7b6054dfadf49af920977d3bd4f8